testutil/pki/serviceaccount.json

```diff
@@ -0,0 +1,7 @@
+{
+  "login_endpoint": "http://127.0.0.1:8101/acs/api/v1/auth/login",
+  "public_key": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmnQ2OJpbQ7RndO6UZ5ic\nnv4logC/G6Mc1xPEofklPyNEesFMq628gmG5aGDmLrFSuztUfb0QMp89649J1iFg\n1EvpnZ0j0Xcz9vzSzf446sqzj+hKqWesjzanx4RDyt8JdOw/M/wTvH1QWUwsUfHH\nslDShwWdBQDw5gqTfSv6MXOzGydHxPZ8YJTFms4AUvUQ/X9Hd93bl0Z/DakHIRbO\nqOKK+qCg+Q0ELarGclCxateUPwnok0hFVCw/5Ok9zer14HxNaXNVuwKKTiScM1Ms\n2PNzfax0+/MMr+dM9ekiXKFh5bRbMriD9yGqbpFH3He1RWfsR1gnROQdrfMjZ/P9\neQIDAQAB\n-----END PUBLIC KEY-----\n",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCadDY4mltDtGd0\n7pRnmJye/iWiAL8boxzXE8Sh+SU/I0R6wUyrrbyCYbloYOYusVK7O1R9vRAynz3r\nj0nWIWDUS+mdnSPRdzP2/NLN/jjqyrOP6EqpZ6yPNqfHhEPK3wl07D8z/BO8fVBZ\nTCxR8ceyUNKHBZ0FAPDmCpN9K/oxc7MbJ0fE9nxglMWazgBS9RD9f0d33duXRn8N\nqQchFs6o4or6oKD5DQQtqsZyULFq15Q/CeiTSEVULD/k6T3N6vXgfE1pc1W7AopO\nJJwzUyzY83N9rHT78wyv50z16SJcoWHltFsyuIP3IapukUfcd7VFZ+xHWCdE5B2t\n8yNn8/15AgMBAAECggEAUst3AVKlF58OUl1OClaBpq0ACV6zEyKV5ClBn74JnFXQ\ncKlL6z+frJisJerDmUYO3jUIrrp5g1lffPV7lCCiYksrOQUyXSe0IHMYobnkUuBD\n7yGLXFIp6z9WIxqgqJB05kt5+ejnv4PBXYz6uRaNgOXYZO66ZCND/1yuAUxgur7A\njVV0XnoJ9ZX+yCS+e6ZhZskTQMa5PEJFgA/7h2VRwHA3mRym6zqT9CiJR69M+qsL\nXtXaK5kC0nMnLgwz2I3tZ3qpt1prOE94Wun7GLaNGkameYqxsgSt76JqwP3154Kj\nLumyzhe24OebggX1AG45bUH5o6q+tYAUbnXrlxQozQKBgQDKREC+JeUzw07KwYFg\nQXA/Ptz/ol421CxP2eF4vT2H8RJOMQbF461QX6fEV48K109Mga7g0B8lT0POGsFR\n3kPzDWj7otDsjSyc+omaaAMBcFZJokXLR3JAkMmM+N3Exyj0ha3yT9FaL7j+3kGj\ny8vCVh48KroZyNaMme8JxyXXowKBgQDDfFDOZYxUNfGhSlp85gB1KXcejdw0AMae\nYpK8UxjVHYLdG00YHPLemJKRAixSBnwadYTDU6VsgfBPU/NW3hXGHukxGdCOKaBC\nukFTX+BDp18YNNzInITR8ukezddaBF/x83MNx0JyoyNLePtjZblZzynsstf3k/pQ\nnuHMvi5YMwKBgG80l2Pz8dBD0U4olbkWQWCEb30dCC+c97/vSsfXQ706DZZv3glu\n5BsMfHC2sy6Cslad9kgkJPueWOk4TJF2YmV/Ofkfyz5DYAPuuzjpdr2Txd02Gqlo\n3qgg8q9PJJ7L4PiPhtNyjufG1S9Lg0tLAPwX3Qe3dho3Eb6743C8zVGBAoGBALJx\nFRYCbiEkiKdxRr5B+WL5ccnLNpmlp6hGVKlhtQY8hQst86ayLiRw40FI2Yoab0h7\nw6/E9/jfd6wB9ttxqDCMYNK4PL64Upj2idEAcKdw7YYcIvlgnoFDGfdLcGu0GJy0\nXG7UV9xq/DnVeqvV3GaVizCMZEWTp2fWUEljxMwvAoGBAIxb9w4uurvh1WbXtVsP\n0bS3Cl1MFxkQuPFPabm2nBzN1aDXXoWBeOSEBt9mK8QGBLdgAwpMnStbg090RW3K\niU9exmKRdLxVpL1eHubIeKdlL5P9DYkHx2/Gw6I3iO6S3UNqRBOfewjedAQWmP+M\nHI5WXq+H/qAwdB29wsweNCJh\n-----END PRIVATE KEY-----\n",
```

Comment:
Nice! copied from the dcos-go test service account? 👏

Reply:
stolen from https://github.com/dcos/dcos-go/tree/master/dcos/http/transport/fixtures 😏